doc/lsst.daf.butler/queries.rst

```diff
@@ -30,7 +30,9 @@ Arguments that specify one or more dataset types can generally take any of the f
  - iterables of any of the above;
  - the special value "``...``", which matches all dataset types.
 
-Some of these are not allowed in certain contexts (as documented there).
+Wildcards (`re.Pattern` and ``...```) are not allowed in certain contexts, such as `Registry.queryDataIds` and `Registry.queryDimensionRecords`, particularly when datasets are used only as a constraint what is returned.
```

Comment:
Triple backticks after ellipsis.
`constraint what is returned` does not quite parse.